src/zappy_server_src/core/argument_handling.c

```diff
@@ -0,0 +1,130 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** argument_handling
+*/
+
+/// \file src/zappy_server_src/core/argument_handling.c
+
+#include "server.h"
+#include "argument_handling.h"
+#include <unistd.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+/// \brief Cross all the args list to detect flags
+/// \param args Argument data structure
+/// \param ac Number of args
+/// \param av List of all the args
+/// \param is_team Is the -n option has been detected ?
+/// \return true When operation succeed
+/// \return false When operation failed
+static bool process_argument_handling(argument_t *args, int ac, char **av,
+bool *is_team)
+{
+    int opt = 0;
+
+    while ((opt = getopt(ac, av, ":p:x:y:nc:f:")) != -1 && opt != '?') {
+        if (opt == 'p')
+            args->port = atol(optarg);
+        if (opt == 'x')
+            args->width = atoi(optarg);
+        if (opt == 'y')
+            args->height = atoi(optarg);
+        if (opt == 'c')
+            args->client_nb = atoi(optarg);
+        if (opt == 'f')
+            args->freq = atoi(optarg);
+        if (opt == 'n')
+            *is_team = true;
+        if (opt == ':')
+            return false;
+    }
```

Comment:
use switch instead of `if if if if if...`

Reply:
In general i agree that switch is better than if, but in this case the use of a switch will exceed the max authorized function length.